datahub-web-react/src/app/search/autoComplete/ParentContainers.tsx

```diff
@@ -1,6 +1,6 @@
 import { FolderOpenOutlined } from '@ant-design/icons';
 import { Typography } from 'antd';
-import React from 'react';
+import React, { Fragment } from 'react';
```

Comment:
Refactored this component a bit so I could give both sets of React fragments unique keys from the urn's.